Cargo.toml

```diff
@@ -29,7 +29,10 @@ exclude = [
 
 [features]
 default = ["compat-3-0-0", "rustls-tls", "dns-resolver"]
-compat-3-0-0 = []
+compat-3-0-0 = ["compat-3-3-0", "bson-2"]
+compat-3-3-0 = []
```

Comment:
Keeping a rolling `compat-VERSION` feature allows us to do this again for other things if we need to, e.g. if we need to go back to supporting multiple async runtimes.

Reply:
Nice. That seems like a good precaution in case there is a future need to make a required feature optional.